Gemfile.lock

```diff
@@ -41,7 +41,7 @@ GEM
     acts_as_votable (0.11.1)
     addressable (2.5.2)
       public_suffix (>= 2.0.2, < 4.0)
-    ahoy_matey (1.6.0)
```

Comment:
Why did you update this file? I know most of them all minor updates but this PR should be focused on **only** adding seeds to populate the homepage